internal/testing/cmp/cmp.go

```diff
@@ -0,0 +1,76 @@
+/*
+ Copyright 2021 - 2022 Crunchy Data Solutions, Inc.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package cmp
+
+import (
+	"strings"
+
+	gocmp "github.com/google/go-cmp/cmp"
+	gotest "gotest.tools/v3/assert/cmp"
+	"sigs.k8s.io/yaml"
+)
+
+type Comparison = gotest.Comparison
+
+// Contains succeeds if item is in collection. The collection may be a string,
+// map, slice, or array. See [gotest.tools/v3/assert/cmp.Contains].
```

Comment:
I would add here the comment in the PR re: why this change for multiline strings

Reply:
🤔 "why this change" ... Do I understand correctly: The godoc here should say that this function does something for multi-line strings.

I'm having a hard time deciding what to add. [gotest.tools/v3/assert/cmp](https://pkg.go.dev/gotest.tools/v3/assert/cmp) doesn't talk about what is in its failure messages.

> Contains succeeds if item is in collection. The collection may be a string, map, slice, or array. See [gotest.tools/v3/assert/cmp.Contains].
>
> When item is missing from collection and both are multi-line strings, … the failure looks like a diff. (?)

Reply:
Yeah, I was thinking of something like the PR comment re: “shows a diff when one multi-line string does not contain another.”

(but also definitely not a sticking point, just something I thought of reading the PR comment and the code comments.)

Reply:
Done in amended https://github.com/CrunchyData/postgres-operator/pull/2981/commits/ea63ca00f3dcbaf0b8c770a7a409cbc4ee13e8c4.